### PR TITLE
Updated generated and SDK for azure autmation based on latest swagger in azure

### PR DESCRIPTION
**Pull request by finiteattractor**
- Regenrated code based on latest automation swagger on azure.
- Updated SDK and test code to address these changes.
Ran all existing test cases and all green.

### CODE REVIEW

File: src/SDKs/Automation/Automation.Tests/Automation.Tests.csproj [hunk 1]
```diff
@@ -11,8 +11,16 @@
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
     <TestProjectType>true</TestProjectType>
   </PropertyGroup>
+
```
Comment (shahabhijeet) [inline src/SDKs/Automation/Automation.Tests/Automation.Tests.csproj]: remove line 12
remove lines 15-19
remove duplicate newtonsoft.json, network package reference

File: src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj [hunk 2]
```diff
@@ -1,6 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <!-- Please do not move/edit code below this line -->
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  
+  <PropertyGroup>
+	<PackageId>Microsoft.Azure.Management.Automation</PackageId>
+    <description>Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.</description>
```
Comment (shahabhijeet) [inline src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj]: Remove lines 5-27
Please include comments above and below line 3 (similar to what you see on line 44, 46)

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
You are missing two meta data files needed for this sdk
Please do the following in below order
1) Please pull latest from upstream
2) execute msbuild build.proj (this will update latest build tools)
3) generate you sdk using generate.cmd (this will add sdkInfo_*.cs file needed for this nuget package)
4) Build you project from command line or ide
this will add AzSdk.RP.props file
5) Update your PR

File: src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -4,16 +4,3 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyTitle("Microsoft Azure Automation Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.")]
-
-[assembly: AssemblyVersion("3.0.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs]: Please include all assembly attributes in the project.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Also clean up your commits, looks like something went wrong the commit history is not clean.
Better create a new PR with clean commit history.

File: src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (vrdmr) [reply src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs]: @shahabhijeet I'll send a new PR with the fixes you requested. @finiteattractor Could you please send your test code and I'll add that to the PR and then we can abandon this one.

**Comment by vrdmr** [comment]
Closing in favor of #3992

**Comment by shahabhijeet** [comment]
@finiteattractor closing this PR as a new one was opened. Please reopen if this is incorrect.

**Pull request closed without merging**